packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.test.tsx

```diff
@@ -13,6 +14,13 @@ function getRowId(row: TableData) {
   return row.id;
 }
 
+function setup(jsx: React.JSX.Element) {
+  return {
+    user: userEvent.setup(),
+    ...render(jsx),
```

Comment:
We never use this as a return right? So can't we just call `render` instead of returning it?